cs-amazon/src/main/java/io/cloudslang/content/amazon/actions/network/CreateSubnetAction.java

```diff
@@ -0,0 +1,154 @@
+package io.cloudslang.content.amazon.actions.network;
+
+import com.hp.oo.sdk.content.annotations.Action;
+import com.hp.oo.sdk.content.annotations.Output;
+import com.hp.oo.sdk.content.annotations.Param;
+import com.hp.oo.sdk.content.annotations.Response;
+import com.hp.oo.sdk.content.plugin.ActionMetadata.MatchType;
+import com.hp.oo.sdk.content.plugin.ActionMetadata.ResponseType;
+import io.cloudslang.content.amazon.entities.constants.Outputs;
```

Comment:
Please use io.cloudslang.content.constants.OutputNames . We should also get rid of the io.cloudslang.content.amazon.entities.constants.Outputs when we have time and refactor all operations to use the cloudslang commons library.